# HG changeset patch
# User jwe@segfault.lan
# Date 1201927959 18000
#      Fri Feb 01 23:52:39 2008 -0500
# Node ID 6499e1ed0bb9ab89fbe766370fb2a7882487918a
# Parent  9f02c32eb70d3e6d0c42a2fd5f3b862f4d6c6bcc
Added tag ss-2-9-18 for changeset 9f02c32eb70d

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -28,8 +28,9 @@ a848b846cb3ac279782b7fef8270a62ca4578539
 3e2cfed6c87e2019cb1584211068b9c581331ccb before-object-branch
 10da0f6d85c260bb5f640436e42d633a57ccda9b ss-2-9-11
 db6d908e8a03a42760a45c7fd29eae074e505f7c ss-2-9-12
 afbd31bb7b4e001c6c16d41eb908aa66bed64352 ss-2-9-13
 fc55a5e1760b832a672d9de3b53888a3d6793d21 ss-2-9-14
 0b91144f9533b05d14fb5692ed67d8c6f500230f ss-2-9-15
 5608935ff5228a90d34fa80dce3dab2462d20fd3 ss-2-9-16
 77de8319c337b39a477d5433348e4597ad61abe0 ss-2-9-17
+9f02c32eb70d3e6d0c42a2fd5f3b862f4d6c6bcc ss-2-9-18
